chisel-wrapper

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eu
+
+CHISEL_BIN="${CHISEL_BIN:-"chisel"}"
+
+CHISEL_CACHE_DIR="$(mktemp -d)"
+CHISEL_DPKG_STATUS_FILE=""
+
+print_usage() {
+	cat <<- EOF
+	Usage: $(basename "$0") [OPTIONS] -- [chisel-cut-OPTIONS] <slice names..>
+
+	This wrapper script performs "chisel cut" operation on specified slices.
+	And if specified in OPTIONS, it performs various tasks such as generating
+	a dpkg status file for the installed file system.
+
+	Every argument after -- will be passed to "chisel cut".
+
+	[OPTIONS]
+
+	  -g, --generate-dpkg-status <path>
```

Comment:
y let's just drop `-g`. shorthands should be minimally intuitive and this one isn't cause it is a very tailored option.